src/repart/repart.c

```diff
@@ -7487,7 +7510,41 @@ static int context_fstab(Context *context) {
                 }
         }
 
-        r = flink_tmpfile(f, t, path, 0);
+        switch (arg_append_fstab) {
+        case APPEND_AUTO:
+                fputs(AUTOMATIC_FSTAB_HEADER_END "\n", f);
+
+                r = read_full_file(path, &c, &l);
+                if (r < 0) {
+                        if (r == -ENOENT) {
+                                log_warning("File fstab not found in %s", path);
```

Comment:
this hence shouldn't be a warning, but just a debug msg.

Reply:
Done